tools/fuzzer_corpus.cc

```diff
@@ -386,36 +388,38 @@ int main(int argc, const char** argv) {
             }
             for (uint32_t num_frames : {1, 3}) {
               spec.num_frames = num_frames;
-
+              for (uint32_t preview : {0, 1}) {
```

Comment:
we are already generating many many inputs, which takes time and might not improve the coverage. Given that keep orientation is pretty independent from other features, would it make more sense to set this as a random value rather than generating every input with and without this? It is actually not an encoder settings so it won't change the testcase (only the last bit).

Reply:
There are two changes in this PR:
 - test "keep orientation"; it uses one bit of flags;
 - add samples with preview images; preview images are already tested, but those are difficult to be created from scratch by fuzzer; we employ encoder param (see line 251) to generate it; orientation was here before this PR; not sure how to setup github to "ignore" spaces to make that visible...

Reply:
There is actually setting to hide whitespace changes (under the cog button).

Reply:
Sorry I missed the preview part I was focused on the keep orientation.